### DISCUSSION TITLE
Logs - "Show Context" read direction

### DISCUSSION THREAD

**Discussion by zackman0010** [grafana/grafana#59353] [Observability]
Currently, the "Show Context" button will print the context in the direction matching the current "Display results" setting, newer results up top or older results up top. However, the opinion of myself and some of my coworkers is that this is counter-intuitive. When we search, we like to have newer results up top for more relevant data to be accessible faster. However, when viewing context, it makes more sense to read top-to-bottom as if we were reading the actual log file. The current behavior makes the context read bottom-to-top, backwards from what we would expect.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
